internal/cluster/validations/validation_test.go

```diff
@@ -246,49 +246,49 @@ var _ = Describe("URL validations", func() {
 
 	Context("test no-proxy", func() {
```

Comment:
Consider using different context for each ocp version

Reply:
done